draft-ietf-rtcweb-jsep.xml

```diff
@@ -5886,6 +5886,12 @@ a=msid:-
 
           <t>If the packet is of type APP, deliver a copy of the packet
           to each "m=" line.</t>
+
+          <t> Some multiparty RTP topologies result in endoints
+          receiving RTCP reports for an SSRC that is not associated
+          with the endpoint that received the RTCP packet. These
+          packets MAY be discarded. </t>
```

Comment:
I don't know - perhaps some things (not browsers) might use them to keep stats on the whole conference or something.